opt_out_http_api/api_methods.py

```diff
@@ -29,10 +32,16 @@ def get_opt_out(self, addresstype, address):
             return opt_outs[0]
         return None
 
-    @app.handle_errors(OptOutNotFound)
-    def opt_out_not_found(self, request, failure):
-        return self.response(
-            request, status_code=404, status_reason="Opt out not found.")
+# Save Opt Out Address
+
+    def save_opt_out(self, addresstype, address):
+        opt_out = {
+            "id": "6666",
```

Comment:
We should generate a unique id here. I would suggest using python's ``uuid`` module, e.g.:
```python
import uuid
print str(uuid.uuid4())
```